src/test/java/com/goterl/lazycode/lazysodium/AEADTest.java

```diff
@@ -122,8 +211,22 @@ public void encryptAES() {
         }
     }
 
+    @Test(expected = AEADBadTagException.class)
+    public void encryptAESMalformedCipher() throws AEADBadTagException {
+        if (lazySodium.cryptoAeadAES256GCMIsAvailable()) {
```

Comment:
Hi, because of this if statement, wouldn't this fail if `cryptoAeadAES256GCMIsAvailable` is false because it expects an exception (`@Test(expected = AEADBadTagException.class)`) all the time?